bpf/bpf_lxc.c

```diff
@@ -51,8 +51,10 @@
 #include "lib/nodeport.h"
 #include "lib/policy_log.h"
 
-#if !defined(ENABLE_HOST_SERVICES_FULL) || defined(ENABLE_SOCKET_LB_HOST_ONLY)
-# define ENABLE_PER_PACKET_LB
+#if !defined(ENABLE_HOST_SERVICES_TCP) || \
+  !defined(ENABLE_HOST_SERVICES_TCP) || \
```

Comment:
`!defined(ENABLE_HOST_SERVICES_TCP)` should this say `!defined(ENABLE_HOST_SERVICES_UDP)` otherwise it's a dup?

Reply:
Yes, it was a copy&paste error, thanks for noticing!